src/runtime/propensity.ts

```diff
@@ -69,22 +71,20 @@ export class Propensity {
     this.propensityServer_.sendSubscriptionState(state, productsOrSkus);
   }
 
-  /** @override */
-  getPropensity(type) {
-    if (type && !Object.values(PropensityApi.PropensityType).includes(type)) {
+  getPropensity(type?: PropensityType): Promise<PropensityScore | null> {
+    if (type && !Object.values(PropensityType).includes(type)) {
       throw new Error('Invalid propensity type requested');
     }
     if (!type) {
-      type = PropensityApi.PropensityType.GENERAL;
+      type = PropensityType.GENERAL;
     }
     return this.propensityServer_.getPropensity(
       this.win_.document.referrer,
```

Comment:
Similarly, I think this requires `this._win` to be set?

Reply:
Yep, it's a `parameter property`